client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -489,6 +492,45 @@ public CompletableFuture<Void> cleanUploadSession(SidecarInstance instance, Stri
                                                             .build());
     }
 
+    /**
+     * Lists CDC commit logs in CDC directory for an instance
+     * @param sidecarInstance instance on which the CDC commit logs are to be listed
+     * @return a completable future with List of cdc commitLogs on the requested instance
+     */
+    public CompletableFuture<ListCdcSegmentsResponse> listCdcSegments(SidecarInstance sidecarInstance)
+    {
+        return executor.executeRequestAsync(requestBuilder()
+                .singleInstanceSelectionPolicy(sidecarInstance)
+                .request(new ListCommitLogsRequest())
+                .build());
```

Comment:
align the chained invocations 

```suggestion
        return executor.executeRequestAsync(requestBuilder()
                                            .singleInstanceSelectionPolicy(sidecarInstance)
                                            .request(new ListCommitLogsRequest())
                                            .build());
```